docs/project/2024/rares.negrila/index.md

```diff
@@ -0,0 +1,101 @@
+# Laser Piano
+A piano made using laser diodes and laser receivers.
+
+:::info 
+
+**Author**: Negrila Rares \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-NegrilaRares
+
+:::
+
+## Description
+
+A piano made using laser diodes and laser receivers.
+
+It uses a potentiometer with leds to adjust and show volume, 
+3 separete switches with leds to displat change in octave 
+and 7 laser diodes and laser receivers that act as the piano keys.
+
+
+The plan is to take the sound data from a microsd card and give it to a dac that will give the analog output to a speaker.
+It uses an LM386 audio amplifier module to keep the signal clear.
+
+
+## Motivation
+
+The reason I chose this project is because I took an interest in laser diodes 
+and wanted to learn file reading/writing from an external memory device 
+as well as understand the conversion of digital code into analog sound.
+
+## Architecture 
+
+<!-- Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
+ -->
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Detail in a few words the hardware used.
+
+### Schematics
+
```

Comment:
Schematic is good but, you should also add a simple [block diagram](https://www.eleccircuit.com/understanding-electronics-block-diagrams-with-example/).

Reply:
You can draw a simple one, just to identify how it will work.